presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1251,7 +1252,9 @@ private void analyzeHaving(QuerySpecification node, Scope scope)
             return legacyAnalyzeOrderBy(node, sourceScope, outputScope, outputExpressions);
         }
 
-        List<SortItem> items = node.getOrderBy();
+        List<SortItem> items = node.getOrderBy()
```

Comment:
we still must set `analysis.setOrderByExpressions(node, orderByExpressions);` `orderByExpressions` would be empty in this case.